sw/device/silicon_creator/lib/boot_data_functest.c

```diff
@@ -340,7 +340,7 @@ rom_error_t write_page_switch_test(void) {
 }
 
 bool test_main(void) {
-  rom_error_t result = kErrorOk;
+  status_t result = OK_STATUS();
```

Comment:
Is there a particular reason for this change? I'd prefer using `rom_error_t` under `sw/device/silicon_creator` exclusively instead of `status_t` for the sake of clarity since all silicon_creator code uses `rom_error_t`.

Reply:
After migrating `EXECUTE_TEST` into ottf, it deals with `status_t`, which can be thought of as a universal error type across the codebase.  All of the silicon_creator code still uses `rom_error_t`.  Silicon creator _tests_ are allowed to use things from elsewhere in the codebase (logging, printf, testutils, etc).

Since `EXECUTE_TEST` converts the result of the executed test functions from `rom_error_t` into `status_t`, the `result` argument to EXECUTE_TEST needs to be `status_t`.

Reply:
> Since `EXECUTE_TEST` converts the result of the executed test functions from `rom_error_t` into `status_t`, the `result` argument to EXECUTE_TEST needs to be `status_t`.

Ah, I missed that the conversion was performed in the macro. SG.